components/Main/Pages/TimerPage.js

```diff
@@ -53,6 +74,19 @@ const styles = StyleSheet.create({
     fontFamily:'GillSans-Light',
     fontSize:40,
     color:'#83ACB2'
+  },
+  PopUpImage: {
```

Comment:
Do not use absolute value when defining the size of the screen. Do it proportional to the screensize to ensure that the proportion maintains with different screensize